src/main/java/edu/greenblitz/robotName/subsystems/arm/ElbowUtils/FalconElbow/FalconElbow.java

```diff
@@ -0,0 +1,69 @@
+package edu.greenblitz.robotName.subsystems.arm.ElbowUtils.FalconElbow;
+
+import com.ctre.phoenix6.controls.MotionMagicDutyCycle;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.greenblitz.robotName.subsystems.arm.ElbowUtils.ElbowConstants;
+import edu.greenblitz.robotName.subsystems.arm.ElbowUtils.ElbowInputsAutoLogged;
+import edu.greenblitz.robotName.subsystems.arm.ElbowUtils.IElbow;
+
+public class FalconElbow implements IElbow {
+
+    private final TalonFX motor;
+
+    public FalconElbow() {
+        motor = new TalonFX(FalconElbowConstants.MOTOR_ID);
+        motor.getConfigurator().apply(FalconElbowConstants.MOTION_MAGIC_CONFIGS);
+        motor.getConfigurator().apply(FalconElbowConstants.CURRENT_LIMITS_CONFIGS);
+        motor.getConfigurator().apply(FalconElbowConstants.CLOSED_LOOP_RAMPS_CONFIGS);
+        motor.getConfigurator().apply(FalconElbowConstants.SWITCH_CONFIGS);
+        motor.setNeutralMode(FalconElbowConstants.NEUTRAL_MODE_VALUE);
+        motor.optimizeBusUtilization();
+    }
+
+    @Override
+    public void setPower(double power) {
+        motor.set(power);
+    }
+
+    @Override
+    public void setVoltage(double voltage) {
+        motor.setVoltage(voltage);
+
+    }
+
+    @Override
+    public void setIdleMode(NeutralModeValue idleMode) {
+        motor.setNeutralMode(idleMode);
+    }
+
+    @Override
+    public void resetAngle(double position) {
+        motor.setPosition(position);
+    }
+
+    @Override
+    public void moveToAngle(double goalAngle) {
+        motor.setControl(new MotionMagicDutyCycle(
```

Comment:
create a `MotionMagicDutyCycle` at the top of the class and every time you want to apply the duty do mmdc.withPosition()